conjur/controller/init_controller.py

```diff
@@ -97,23 +98,27 @@ def get_server_certificate(self):
 
         return fetched_certificate
 
+    # pylint: disable=line-too-long,logging-fstring-interpolation,broad-except,raise-missing-from
     def get_account_info(self, conjurrc_data):
         """
         Method to fetch the account from the user
         """
         if conjurrc_data.account is None:
             try:
                 self.init_logic.fetch_account_from_server(self.conjurrc_data)
-            # pylint: disable=broad-except,logging-fstring-interpolation
+            except CertificateHostnameMismatchException:
+                raise
             except Exception as error:
-                # pylint: disable=line-too-long,logging-fstring-interpolation
-                logging.warning(f"Unable to fetch the account from the Conjur server. Reason: {error}")
-                # If there was a problem fetching the account from the server, we will request one
-                conjurrc_data.account = input("Enter the Conjur account name (required): ").strip()
-
-                if conjurrc_data.account is None or conjurrc_data.account == '':
-                    # pylint: disable=raise-missing-from
-                    raise RuntimeError("Error: account is required")
+                # Check for catching if the endpoint is exists. If the endpoint does not exist,
+                # a 401 status code will be returned.
+                # If the endpoint does not exist, the user will be prompted to enter in their account.
+                # pylint: disable=no-member
+                if hasattr(error.response, 'status_code') and str(error.response.status_code) == '401':
+                    conjurrc_data.account = input("Enter the Conjur account name (required): ").strip()
+                    if conjurrc_data.account is None or conjurrc_data.account == '':
+                        raise RuntimeError("Error: account is required")
```

Comment:
@sigalsax is this the message? 
if so i think we need to provide more data to the user - something like
"please use --account flag to provide it."

Reply:
This is not the flow when this error will be met. This flow is for when aren't able to get the account (running against a Conjur OSS server) and when we prompt them for their account, give us an empty value then we error